pkg/codegen/testing/test/testdata/regress-py-12980/python/pulumi_myPkg/mymod/childb/member_b2.py

```diff
@@ -0,0 +1,116 @@
+# coding=utf-8
+# *** WARNING: this file was generated by test. ***
+# *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import copy
+import warnings
+import pulumi
+import pulumi.runtime
+from typing import Any, Mapping, Optional, Sequence, Union, overload
+from ... import _utilities
+from . import outputs
+from ... import mymod
+from ... import mymod as _mymod
+from ..childa.member_a1 import MemberA1
```

Comment:
I haven't dug in enough to know if it's related to this change, but `MemberA1` isn't referenced anywhere I can see. Below it's referenced as `_mymod.childa.MemberA1`.

Reply:
Good catch. I went back and forth a bit on this, and I think when importing something from another module, we should always use qualified names (`mymod.childa.MemberA1`) to avoid any potential name clashes if we have the same member name in different modules.

I updated the code to generated an import for this (`from ... import mymod` in this case) and only generate local/relative imports when importing a member from the same module.